stock_ownership_by_move/view/picking.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="view_picking_form" model="ir.ui.view">
+      <field name="name">view_picking_form</field>
+      <field name="model">stock.picking</field>
+      <field name="inherit_id" ref="stock.view_picking_form"/>
+      <field name="arch" type="xml">
+
+        <xpath expr="//field[@name='owner_id']/.." position="replace"/>
```

Comment:
@yvaucher I tried this https://github.com/lepistone/stock-logistics-workflow/commit/af97a90163155fb4e75959f1db1a98c8da47c269 (with `eval`) but it doesn't work (I still see the owner).